src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -407,9 +409,10 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
         resizeable: false,
         sortable: false,
         draggable: false,
-        checkboxable: true,
+        checkboxable: false,
```

Comment:
When checkboxable is true,  ngx-datatable appends the checkbox component used for row selection. 
 
![image](https://user-images.githubusercontent.com/15856785/178557566-90c9138a-db46-4117-b761-0886136352dc.png)

Since we are using a custom checkbox now, leaving it set to true results in two checboxes being rendered instead. I found the custom checkbox works fine and  was actually thinking we could remove this line entirely